src/manage_nsfs/manage_nsfs_help_utils.js

```diff
@@ -281,6 +281,42 @@ Usage:
 
 `;
 
+
+const UPGRADE_OPTIONS = `
+Usage:
+
+'upgrade' is a noobaa-core command that will return the exported metrics of the deployed NooBaa system.
+
+Usage:
+
+    noobaa-cli upgrade <action> [flags]
+
+List of actions supported:
+
+    start
+    status
+    history
+    
+`;
+
+const UPGRADE_START_OPTIONS = `
+'upgrade start' is a noobaa-core command that will start config directory upgrade run.
```

Comment:
Should the cluster admin run it only in one of the nodes? (the config structure is mutual for all the nodes).
Worth mentioning it.